src/app/static/features/features.component.ts

```diff
@@ -1,16 +1,17 @@
 import { Component, OnInit } from '@angular/core';
 
-import { environment as env } from '@env/environment';
 import { ROUTE_ANIMATIONS_ELEMENTS } from '@app/core';
 
+import features from './features';
```

Comment:
Maybe `import { features } from './features-data'` or features-list or something?

and then in comp `features = features` or `features = featuresData`. Just because the `features = features.features;` is a bit goofy 😹

Also maybe we can provide `export interface Feature {}` with mandatory and optional fields so that it is easy to add more for the feature contributors ;)

Reply:
Totally agreed, this one should be cleaner. I just found the way to export data as typed const instead of default.